crt_portal/cts_forms/templates/partials/policy.html

```diff
@@ -1,30 +0,0 @@
-{% load i18n %}
-
-<div class="grid-container">
-  <div class="grid-row">
-
-    <div class="grid-col-12">
-      <div class="usa-footer__heading usa-footer__logo-heading">
-        Privacy Policy
-        <div class="usa-footer__separator"></div>
-      </div>
-
-      <div class="grid-row grid-gap">
-        <div class="grid-col-9 usa-footer__links" style="font-size: 14px">
-          <p style="margin-bottom: 1em">
-            The purpose of this form is to allow the public to submit civil rights complaints to the Department of Justice, thereby allowing us to enforce over thirty civil rights statutes within our authority. <a style="color: white; text-transform: none" href="https://www.justice.gov/crt/privacy-policy">These statutes</a> authorize us to collect this information.  You should know that any information you provide through this form is voluntary, yet failure to provide some of the information might limit the Department’s ability to pursue your claim. We may use this information for certain routine uses, including sharing this information under certain circumstances with:
```

Comment:
Update this link to the new privacy policy

Reply:
✔️